jax_lensing/models/convdae.py

```diff
@@ -189,6 +189,7 @@ def __init__(self,
                use_bn=True,
                pad_crop=False,
                variant='EiffL',
+               deepmass=False,
```

Comment:
Instead of a deepmass flag, we could just check if a condition is supplied when calling the model, and disable the conditioning if that's not the case